charts/apisix/templates/configmap.yaml

```diff
@@ -46,7 +46,11 @@ data:
     {{- else }}
     apisix:    # universal configurations
       {{- if not (eq .Values.deployment.role "control_plane") }}
-      node_listen: {{ .Values.gateway.http.containerPort }}    # APISIX listening port
+      node_listen:    # APISIX listening port
```

Comment:
Could we also support multiple HTTPS ports? Not only for HTTP.

Reply:
emm.. In order not to affect the previous, i added `additionalContainerPorts`